pkg/project/project.go

```diff
@@ -76,6 +67,15 @@ func Delete(client *occlient.Client, projectName string) error {
 		return errors.Wrap(err, "unable to delete project")
 	}
 
+	// If there will be any projects post the current deletion,
+	// Choose the first project from remainder of the project list to set as current
+	if len(projects) > 0 {
+		currentProject = projects[0].Name
```

Comment:
Ack added

Reply:
Hmm but that would add a latency of probably 10-12 seconds.
For some strange reason, delete of a resource takes some time to reflect in the list of resources..
Will add it though

Reply:
am confused
why does it take 10sec to switch project and print info to terminal?

Reply:
No I meant:

10 seconds is not for switching project it is for the delete of a project to reflect in project list only after which I show current project.. And its not exactly 10 seconds but around 5-6 seconds so, if we were to show the current project, the command would have to wait/hang for that amount of period(not relying on the time in seconds anymore but watching the project list now... instead of hard coding the wait time interval based on a comment thread as in https://github.com/redhat-developer/odo/pull/740#discussion_r221626756) ...